qa/workunits/ceph-helpers.sh

```diff
@@ -1282,6 +1302,70 @@ function test_display_logs() {
 }
 
 #######################################################################
+##
+# Spawn a command in background and save the pid in the variable name
+# passed in argument
+#
+# Example:
+#   pids1=""
+#   run_in_background pids1 bash -c 'sleep 1; exit 1'
+#
+# @param pid_variable the variable name (not value) where the pids will be stored
+# @param $@ the command to execute
```

Comment:
**$@** also includes the first argument, use **...** instead (that's how it's done in other functions as well, C prototype style).